edunext_openedx_extensions/ednx_microsites/backends/database.py

```diff
@@ -12,7 +12,7 @@
     from microsite_configuration.backends.base import BaseMicrositeBackend
 except ImportError:
     from .base import BaseMicrositeBackend
-from microsite_configuration.models import Microsite
+from ..models import Microsite
```

Comment:
I think we should stick to absolute imports. It is the default in python 3 and the preferred way to do imports in the community and pep-328. For the scope of this PR, I say we should only change it here, and later we will make a different PR changing all the remaining relative imports.